fiware/doc/demo.md

```diff
@@ -194,5 +180,4 @@ curl 192.168.1.59:1026/v2/entities/hello_ros2?type=std_msgsString -s -S -H 'Acce
 curl 192.168.1.59:1026/v2/subscriptions/5cc3f909efccc70b574a4611 -s -S -H 'Accept: application/json' -X DELETE
 ```
 
-For removing all subscriptions, please, use the script found into utils `erase_subscriptions.sh`
-
+For removing all subscriptions, please, use the script found in fiware/utils called `erase_subscriptions.sh`
```

Comment:
Add: 
**CAREFUL**, this script will delete all your instances in your server.

In fact, I do not know if we should distribute this script, is too dangerous, and a dds-fiware user does not want to erase all his data. The curl part for removing this concrete subscription could be enough.

Reply:
Ok I'm removing it